<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>776</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>1111</width>
      <height>731</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Automatic</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>621</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Playing automatic mode, please follow the instructions:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter starting word:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>190</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter target word:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="startWordEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>180</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="targetWordEdit">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>190</y>
        <width>113</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pathPushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>250</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Calculate Path</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>310</y>
        <width>591</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="browseButton_1">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse Dictionary</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dictionaryPathLineEdit_1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>581</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="statusEdit_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>510</y>
        <width>731</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>520</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Standard</string>
     </attribute>
     <widget class="QPushButton" name="browseButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse Dictionary</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dictionaryPathLineEdit_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>461</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="startWordDisplayLabel">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting Word:</string>
      </property>
     </widget>
     <widget class="QLabel" name="endWordDisplayLabel">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>230</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ending Word:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="userStartWord">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="userEndWord">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>220</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="userInputPath">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>330</y>
        <width>671</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Please enter the path:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>450</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>510</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="statusEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>500</y>
        <width>851</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="startGameButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>140</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="wordSpinBox">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>50</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>50</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Word Length:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hintButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>HINT</string>
      </property>
     </widget>
     <widget class="QLabel" name="hintDisplayLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>410</y>
        <width>801</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="gameSaveButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>590</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Game</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Analytics</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter user name for analytics:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="analyticsUserNameInput">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="analyticsPushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>140</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Analytics</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="analyticsResultsDisplay">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>280</y>
        <width>851</width>
        <height>351</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>pathPushClicked()</slot>
 </slots>
</ui>
